tests/AudioFileTests/WavLoadingTests.cpp

```diff
@@ -95,6 +95,95 @@ BOOST_AUTO_TEST_CASE (WavLoadingTests_Stereo_24bit_44100)
+    { -0.022707462310791   , -0.0217818021774292 },
+    { -0.017827033996582   , -0.0234956741333008 },
+    { -0.00089371204376220 , -0.0021892786026001 },
+    { 0.0107858180999756   ,  0.00936055183410645 },
+    { 0.00766479969024658  ,  0.0019000768661499 },
+    { -0.00178730487823486 , -0.0047607421875 },
+    { -0.00629222393035889 , -0.000453710556030273 },
+    { 0.00178396701812744  ,  0.0112038850784302 },
+    { 0.0313527584075928   ,  0.0283315181732178 },
+    { 0.0207730531692505   ,  0.0117709636688232 },
+    { -0.022290825843811   , -0.0326575040817261 },
+    { 0.00968170166015625  , -0.00387370586395264 },
+    { -0.00373435020446777 , -0.00323045253753662 },
+    { -0.0470277070999146  , -0.0545462369918823 },
+    { 0.0142315626144409   ,  0.00163185596466064 },
+    { 0.0251777172088623   ,  0.0433876514434814 },
+    { -0.00247633457183838 ,  0.0187458992004395 },
+    { 0.0208892822265625   ,  0.0335485935211182 },
+    { -0.0109539031982422  ,  0.00750398635864258 },
+    { -0.0392974615097046  , -0.0354713201522827 },
+    { -0.00938189029693604 , -0.0357832908630371 },
+    { -0.0166007280349731  , -0.0415679216384888 },
+    { -0.053926944732666   , -0.0326246023178101 },
+    { -0.0511249303817749  , -0.0796664953231812 },
+    { 0.012263298034668    ,  0.00228583812713623 },
+    { 0.103702783584595    ,  0.119603276252747 },
+    { 0.105173587799072    ,  0.170729041099548 },
+    { 0.061316967010498    ,  0.133585691452026 },
+    { 0.107999563217163    ,  0.101216077804565 },
+    { 0.0264698266983032   ,  0.123700022697449 },
+    { -0.0883380174636841  ,  0.0142438411712646 },
+    { -0.0407258272171021  , -0.0427062511444092 },
+    { -0.0103473663330078  , -0.00744879245758057 },
+    { -0.00884878635406494 ,  0.0219517946243286 },
+    { 0.078632116317749    ,  0.0263230800628662 },
+    { -0.0113770961761475  ,  0.0128881931304932 },
+    { -0.0830408334732056  , -0.0314878225326538 },
+    { 0.142681479454041    ,  0.0151617527008057 },
+    { 0.0191718339920044   ,  0.00898754596710205 },
+    { -0.0219069719314575  ,  0.00101673603057861 },
+    { 0.0249385833740234   ,  0.0396249294281006 },
+    { -0.0596954822540283  , -0.0273034572601318 },
+    { 0.0529693365097046   ,  0.00769448280334473 },
+    { -0.026525616645813   , -0.0628191232681274 },
+    { 0.0403858423233032   , -0.0374784469604492 },
+    { -0.00207006931304932 , -0.0513983964920044 },
+    { 0.0385066270828247   ,  0.0237759351730347 },
+    { 0.208563089370728    ,  0.180346012115479 },
+    { 0.0995622873306274   ,  0.0979213714599609 },
+    { 0.171254515647888    ,  0.214262962341309 },
+    { 0.149603962898254    ,  0.149571657180786 },
+    { 0.13909387588501     ,  0.128116130828857 },
+    { 0.140897393226624    ,  0.241058707237244 },
+    { 0.0962514877319336   ,  0.143865942955017 },
+    { 0.0498672723770142   ,  0.0476465225219727 },
+    { 0.0204880237579346   ,  0.0452275276184082 },
+  };
+
```

Comment:
This style of "embedded" test data is maybe not the nicest way... it does bloat the test code a bit.
Also, I could change this to `LLLLLRRRRR` rather than the current `LRLRLRLRLR` if you prefer.

Reply:
I haven't merged this in yet (mainly just to not having much time) - but when I do I'll take a look at the tests properly :)